platform/lib/routers/healthCheck.js

```diff
@@ -25,7 +26,7 @@ const HEALTH_CHECK_PATH = '/__health-check';
 // Used by GCE to determine wether a VM instance is healthy.
 healthCheck.get(HEALTH_CHECK_PATH, (req, res) => {
   // TODO add more checks
-  console.log('[HEALTH CHECK] OK');
+  log.info('[HEALTH CHECK] OK');
```

Comment:
This would log `[4:50:43 PM] [Health Check] › ℹ  info      [HEALTH CHECK] OK`, so you can remove the manual scope infront of `OK`.